client/my-sites/checkout/checkout-thank-you/domains/thank-you-content/index.tsx

```diff
@@ -33,9 +30,9 @@ interface StepCTAProps {
 const StepCTA = ( { email, primary, siteName }: StepCTAProps ) => {
 	const titanAppsUrlPrefix = useTitanAppsUrlPrefix();
 
-	const redirectUrl = `${ window.location.protocol }//${
-		window.location.host
-	}${ emailManagementMailboxes( siteName ) }`;
+	const redirectUrl = `${ window.location.protocol }//${ window.location.host }${ getMailboxesPath(
+		siteName
+	) }`;
```

Comment:
This could be simplified to:
```js
const redirectUrl = new URL( getMailboxesPath( siteName ), window.location ).href;
```
Constructs from an incomplete URL (path only) and uses `window.location` as the base. It also has the benefit of including the `window.location.port` automatically, like in `http://calypso.localhost:3000`.

(this code originally added in #65087)